helm/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "display_name": "Helm",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "helm",
+  "metric_prefix": "helm.",
+  "metric_to_check": "helm.release",
+  "creates_events": false,
+  "short_description": "Track your Helm deployments with Datadog",
+  "guid": "7f57a1f2-6dc6-4029-ae9c-426238ea2f86",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Helm",
+  "categories": [
```

Comment:
You can add the datadog integrations categories here? The categories are the ones seen in the integrations doc page: https://docs.datadoghq.com/integrations/

Reply:
Added :+1: